lib/approval_request_listener.rb

```diff
@@ -0,0 +1,64 @@
+require "manageiq-messaging"
+
+class ApprovalRequestListener
+  SERVICE_NAME = "platform.approval".freeze
+  CLIENT_AND_GROUP_REF = "catalog-api-worker".freeze
+
+  class ApprovalRequestNotFound < StandardError; end
```

Comment:
should this be moved to `lib/exceptions.rb`? That's where we have the rest of the exception definitions.

Reply:
I don't know if the `ApprovalRequestNotFound` really needs to be in the general "catalog" scope, it's only specific to this class and I don't think it really has a use outside of it.

Reply:
good point, it's a very specific exception.